docs/source/_templates/components.rst

```diff
@@ -0,0 +1,47 @@
+{% if current_project != 'the labscript suite' %}
+.. toctree::
+    :maxdepth: 2
+    :hidden:
+
+    The labscript suite <{{intersphinx_mapping['labscript-suite'][0]}}>
```

Comment:
`the *labscript suite*`, or just `*labscript suite*`?

Reply:
What I have now matches the docs project name and [index page headers](https://docs.labscriptsuite.org/en/latest/) (minus the italics which I'm not sure we can do easily)

Reply:
Project name is all lowercase.

Reply:
Should "The" be capitalised in the project name then? Seems like it should be (otherwise wouldn't we be writing "*the labscript suite*" everywhere rather than "the *labscript suite*"?)

Reply:
> Should "The" be capitalised in the project name then? Seems like it should be

No, all lowercase<sup>†</sup> "the _labscript suite_" is a deliberate style/branding choice. 

> otherwise wouldn't we be writing "the labscript suite" everywhere rather than "the labscript suite"?

Not necessarily, as the article _labscript suite_ is sometimes used as a proper noun in isolation from the definite article throughout our material.

† In titles and mid-sentence, i.e. sentences beginning with this read "The _labscript suite_ ...".

Reply:
Done.